src/pages/Gallery/components/PictureList/PictureList.tsx

```diff
@@ -1,12 +1,33 @@
-import { PictureItem } from '../PictureItem'
+import { useGetNASAImages } from '@/services/queries/gallery'
+import { useEffect, useState } from 'react'
+import { PictureItem, SkeletonPictureItem } from '..'
+import { CATEGORY_MAP, NASA_IMAGE_BASE } from '../../gallery.constants'
 import * as S from './PictureList.styles'
 import type { PictureListProps } from './PictureList.types'
 
-export const PictureList = ({ pictures, onImageClick }: PictureListProps) => {
+export const PictureList = ({ onImageClick, selectedCategory }: PictureListProps) => {
+  const { data } = useGetNASAImages({ q: CATEGORY_MAP[selectedCategory] })
+
+  const [isEachImgLoaded, setIsEachImgLoaded] = useState<Array<boolean>>(Array(16).fill(false))
+
+  useEffect(() => {
+    setIsEachImgLoaded(Array(16).fill(false))
+  }, [selectedCategory])
+
+  const pictures = data?.collection.items
+
   return (
     <S.PictureListContainer>
-      {pictures.map((picture) => (
-        <PictureItem key={picture.links[0].href} picture={picture} onImageClick={onImageClick} />
+      {Array.from({ length: 16 }).map((_, index) => (
+        <S.PictureOneWrapper key={selectedCategory + index}>
+          {!isEachImgLoaded[index] && <SkeletonPictureItem />}
+          <PictureItem
+            index={index}
+            picture={pictures ? pictures[index] : NASA_IMAGE_BASE}
+            onImageClick={onImageClick}
+            setIsEachImgLoaded={setIsEachImgLoaded}
+          />
```

Comment:
전 사실 이전부터 `NASA_IMAGE_BASE`가 굳이 필요할까라는 생각을 많이 했었는데요~!

아예 존재자체 없애버리고 
```suggestion
          {pictures && <PictureItem
            index={index}
            picture={pictures[index]}
            onImageClick={onImageClick}
            setIsEachImgLoaded={setIsEachImgLoaded}
          />}
```
로 해도 동일하게 잘 동작하는 거 같습니다~

Reply:
- [refactor: 불필요한 상수 제거, 해당 상수에 의존한 코드 수정](https://github.com/2023-chambit-project/KOOSMOOS/pull/78/commits/8e46ea96ae04440a6cdd4d2c2de830700ea14081)
  - 오.. 진심 감탄합니다.. 반영했습니다.!